bigframes/dataframe.py

```diff
@@ -906,16 +906,18 @@ def drop(
                 columns = labels
 
         block = self._block
-        if index:
+        if index is not None:
             level_id = self._resolve_levels(level or 0)[0]
 
-            if utils.is_list_like(index):
+            if utils.is_list_like(index) or isinstance(index, pandas.Index):
```

Comment:
Doesn't utils.is_list_like already match indices?

Reply:
good point, only the fix above ("if index is not None:") is needed. This line is not.